composer.json

```diff
@@ -37,22 +37,23 @@
         }
     ],
     "require": {
-        "php": ">=7.2 < 8.3",
-        "psr/log": "^1.1.4",
+        "php": ">=8.0",
```

Comment:
`>=8.0 < 8.3` is more appropriate I think. We're not testing other PHP versions, so we should be clear about what we support. PHP 8.3 is out in a couple of weeks.

Reply:
done